diff --git a/dist/module.js b/dist/module.js
index 80231f09c1c022da3d366b39bb2ea31c2555ff98..e84a1f11333eefcf6d9479aa6a928408c080ee2a 100644
--- a/dist/module.js
+++ b/dist/module.js
@@ -377,11 +377,13 @@ function pick(obj, keys) {
   return newobj;
 }
 function generateImageOptions(providers, imageOptions) {
+  const imageOptionsWithoutProvider = { ...imageOptions };
+  delete imageOptionsWithoutProvider.provider;
   return `
   ${providers.map((p) => `import ${p.importName} from '${p.runtime}'`).join("\n")}
   
   export const imageOptions = {
-    ...${JSON.stringify(imageOptions, null, 2)},
+    ...${JSON.stringify(imageOptionsWithoutProvider, null, 2)},
     /** @type {${JSON.stringify(imageOptions.provider)}} */
     provider: ${JSON.stringify(imageOptions.provider)},
     providers: {
